aries-site/src/examples/foundation/typography/HeadingSizingExample.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Box, Heading } from 'grommet';
+
+export const HeadingSizingExample = () => {
+  return (
+    <Box direction="column" gap="large">
+      <Box direction="column" gap="medium">
+        <Heading level={1} size="medium" margin="none">
```

Comment:
I think `medium` is the default so mentioning the size repeatedly will be redundant.

Reply:
Agree that it is redundant.  My thought was that since this is being used as an example on the site, I had added the `size="medium"` attempting to be explicit -- trying to reinforce the connection between medium = default. Helpful? Detrimental? Thoughts?